### ISSUE TITLE
[Feature Request] [Urgent] Update Redactor framework

### ISSUE THREAD

**Issue by phillf** [osTicket/osTicket#5072]
The current version of Redactor present in the codebase is out of date and should be updated. There are various bug fixes as well  as some accessibility issues (eg. screen reader users getting stuck in text fields or not being able to use user pickers.) that are likely fixed with an update.

Frankly imo, the fact that this plugin, which is essential to the operation of OST, is as far out of date as it has been allowed to be is ridiculous.

**Comment by JediKev** [comment]
@phillf

You should probably do some searching before creating an issue as you would’ve seen that updated Redactor (latest version) is already released as part of v1.14-rc1. 

And it’s not quite that simple...v1.12.2 and below uses the latest FREE version of Redactor as it went closed source a few years ago. We had to aquire licenses so that’s why we are able to update it now and distribute it. Also, the entire Redactor codebase changed when they went closed source so we had to rewrite EVERYTHING with the editor.

Cheers.
